src/primitives/polyline/styled.rs

```diff
@@ -45,6 +45,21 @@ where
             Rectangle::new(self.primitive.bounding_box().center(), Size::zero())
         }
     }
+
+    fn draw_thick<D>(&self, stroke_color: C, display: &mut D) -> Result<(), D::Error>
```

Comment:
This is more of general pedantic nit and not specific to this PR:
Can we decide on one name for the `DrawTarget` arguments? `target` and `display` are used in very similar quantities. My vote would be `target`, because it's more generic, but I don't really care as long as it's consistent.

Reply:
`target` makes most sense to me too. I'll try and adhere to that name from now on - consistency is indeed important.